lib/fulfillment_resolver.js

```diff
@@ -0,0 +1,56 @@
+const { isInRecap } = require('./util')
+
+class FulfillmentResolver {
+  constructor (responseReceived) {
+    this.elasticSearchResponse = responseReceived
+  }
+  responseWithFulfillment () {
+    for (let hit of this.elasticSearchResponse.hits.hits) {
+      (hit._source.items || []).map((item) => {
+        if (item.electronicLocator) return item
+        const _isInRecap = isInRecap(item)
+        item.physFulfillment = this._determinePhysFulfillment(item, _isInRecap, false)
+        item.eddFulfillment = this._determineEddFulfillment(item, _isInRecap, true)
```

Comment:
The `*Fulfillment` properties are references to other models, so should use `@id` like:
```
  "physFulfillment": { "@id": "fulfillment:sasb-onsite" }
```

Reply:
should it also be an array?